### PR TITLE
settling for 7.6.1, serious compiler issues for latest version

### PR DESCRIPTION
**Pull request by doutriaux1**
no make check step in this library

### CODE REVIEW

**Comment by doutriaux1** [comment]
@ocefpaf i think it's ready for feedstock, do i need todo anything else? Thanks!

File: recipes/mesalib/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include "${CFLAGS}
+export LDFLAGS="-L${PREFIX}/lib "${LDFLAGS}
+
+./configure  --prefix=$PREFIX --with-driver=osmesa --disable-egl --with-osmesa-bits=32
+make -j${CPU_COUNT}
+make install
```
Comment (ocefpaf) [inline recipes/mesalib/build.sh]: Is there a `make check` we can run here?

File: recipes/mesalib/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include "${CFLAGS}
+export LDFLAGS="-L${PREFIX}/lib "${LDFLAGS}
+
+./configure  --prefix=$PREFIX --with-driver=osmesa --disable-egl --with-osmesa-bits=32
```
Comment (ocefpaf) [inline recipes/mesalib/build.sh]: Can you comment on the `32`?

File: recipes/mesalib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{% set name = "MesaLib" %}
+{% set version = "7.6.1" %}
+package:
```
Comment (ocefpaf) [inline recipes/mesalib/meta.yaml]: Add a line space here.

File: recipes/mesalib/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: ftp://ftp.freedesktop.org/pub/mesa/older-versions/7.x/{{ version }}/{{ name }}-{{ version }}.tar.gz
+    sha256: e2b5595952e94d5ecdb7c0a53ac6db9bbf23cc2228b09124ae0c37f9507690ac
+
+build:
+    skip: True  # [win]
+    number: 0
+    track_features:
+        - mesalib
+requirements:
```
Comment (ocefpaf) [inline recipes/mesalib/meta.yaml]: Please add line space between every new section in the `meta.yaml` fo readability purposes.

File: recipes/mesalib/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  build:
+    - python <3
```
Comment (ocefpaf) [inline recipes/mesalib/meta.yaml]: Use only `python` and move the version restriction to the `skip` statement: `skip: True  # [win or py3k]`

File: recipes/mesalib/build.sh (hunk 2 above)
Comment (doutriaux1) [inline recipes/mesalib/build.sh]: 32 bit depth for simulated display, default is 8bit.

File: recipes/mesalib/build.sh (hunk 1 above)
Comment (doutriaux1) [inline recipes/mesalib/build.sh]: old code no `make check`

File: recipes/mesalib/meta.yaml (hunk 5 above)
Comment (doutriaux1) [inline recipes/mesalib/meta.yaml]: no can do. Tried it for some reason if I do so, it tells me there's nothing to build. Look at commits history.

**Comment by ocefpaf** [comment]
Done in https://github.com/conda-forge/staged-recipes/pull/2450

**Pull request closed without merging**